core/src/main/scala/org/http4s/headers/Accept.scala

```diff
@@ -1,12 +1,25 @@
 package org.http4s
 package headers
 
+import org.http4s.util.{Renderable, Writer}
+
 import scalaz.NonEmptyList
 
-object Accept extends HeaderKey.Internal[Accept] with HeaderKey.Recurring
+object Accept extends HeaderKey.Internal[Accept] with HeaderKey.Recurring {
+  implicit def withDefaultQValue(mediaRange: MediaRange): MediaRangeAndQValue =
```

Comment:
I think most people are moving away from implicit conversions. We've made a pattern of using overloaded `apply` functions for other situations similar to this and I would probably have tried that first here (can't say it would have been successful :smile:).